contentcuration/contentcuration/static/js/edit_channel/preview/views.js

```diff
@@ -169,8 +171,23 @@ var PreviewView = BaseViews.BaseView.extend({
             } else if (document.msExitFullscreen) {
                 document.msExitFullscreen();
             }
-            this.$(".view_fullscreen").text("Show Fullscreen");
         }
+    },
+    exit_fullscreen:function(){
```

Comment:
https://trello.com/c/asKGwfpR/201-pressing-full-screen-and-then-esc-doesn-t-close-the-full-screen-window